### PR TITLE
Updated examples, version 0.8.0

### PR DESCRIPTION
**Pull request by debris**
- updated examples to create contract asynchronously
- version `0.8.0`

### CODE REVIEW

File: example/contract.html [hunk 1]
```diff
@@ -34,22 +34,27 @@
...
-                // TODO: compiled code do not have 0x prefix
-                return mined || (th.from === web3.eth.defaultAccount && th.input.indexOf(code) !== -1);
-            }, false);
-
-            if (contractMined) {
-                document.getElementById('status').innerText = 'Mined!';
-                document.getElementById('call').style.visibility = 'visible';
+        web3.eth.contract(abi).new({data: code}, function (err, contract) {
+            if (err) {
+                console.error('contract creation failed!');
```
Comment (frozeman) [inline example/contract.html]: You can simple log the error object, it contains the text for the reason, why it failed.

File: example/contract_array.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+        // let's assume that coinbase is our account
+        web3.eth.defaultAccount = web3.eth.coinbase;
+
+        var watch = web3.eth.filter('latest');
+
+        // create contract
+        web3.eth.contract(abi).new({data: code}, function (err, contract) {
+            if (err) {
+                console.error('contract creation failed!');
```
Comment (frozeman) [inline example/contract_array.html]: Same here

File: example/event_inc.html [hunk 3]
```diff
@@ -37,27 +37,24 @@
             
             var watch = web3.eth.filter('latest');
```
Comment (frozeman) [inline example/event_inc.html]: i think this is not necessary anymore

Comment (debris) [inline example/event_inc.html]: indeed

File: example/contract.html [hunk 4]
```diff
@@ -34,22 +34,17 @@
         var watch = web3.eth.filter('latest');
```
Comment (frozeman) [inline example/contract.html]: this is outdated too

**Comment by radster360** [comment]
Thanks. Many desirable changes in this release. Besides fixing the issues, we like the new contract new() calling mechanism with callback to deal with.

**Pull request merged**